From 546d53c3515e8a488a204763437d1fa0917097e5 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 11 Dec 2018 12:39:30 +0100
Subject: [PATCH] Add a define so that gir compilation succeeds

For some reason meson 0.49.0 does not anymore pass global arguments to gir compiler.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 6ff8a6a..f0b2887 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -132,7 +132,7 @@ if get_option('with_introspection')
         install_dir_gir: girdir,
     install_dir_typelib: typelibdir,
         export_packages: libdazzle_package,
-             extra_args: [ '--c-include=dazzle.h', '--quiet' ],
+             extra_args: [ '--c-include=dazzle.h', '--quiet', '-DDAZZLE_COMPILATION' ],
   )
 
   if get_option('with_vapi')
